src/sentry/api/helpers/group_index/update.py

```diff
@@ -734,6 +734,10 @@ def handle_is_bookmarked(
             group__in=group_ids,
             user_id=acting_user.id if acting_user else None,
         ).delete()
+        if features.has("organizations:participants-purge", group_list[0].organization):
+            GroupSubscription.objects.filter(
+                user_id=acting_user.id, group__in=group_ids, reason=GroupSubscriptionReason.bookmark
```

Comment:
I'm intentionally not filtering on `reason=GroupSubscriptionReason.bookmark` because we never update the reason - e.g. if you are assigned to an issue but then also bookmark the issue, the reason will remain "GroupSubscriptionReason.assigned".